requirements.txt

```diff
@@ -0,0 +1,8 @@
+requests
```

Comment:
Yeah, we need to build `requirements.txt` going forward. I will add my modules as well.